openhands/runtime/client/client.py

```diff
@@ -369,7 +370,7 @@ async def run(self, action: CmdRunAction) -> CmdOutputObservation:
                 else:
                     output, exit_code = self._execute_bash(
                         command,
-                        timeout=SOFT_TIMEOUT_SECONDS
+                        timeout=HARD_TIMEOUT_SECONDS
```

Comment:
5 seconds is too short for any arbitrary bash command - let's make it 2 minutes!

Reply:
Any idea why we don't want to raise the timeout for the other two occurrences of `SOFT_TIMEOUT`? 

I think the 5 second here is introduced by @rbren for interactive bash commands - but i do think 5 seconds is definitely too short here

Reply:
Is there any chance we could reasonably use here the config value for `sandbox.timeout`? I'd love to make it 10 seconds on my machine without hacking it 😅

Reply:
(yes that was what i wanted too 🤣 )